index.js

```diff
@@ -24,7 +34,7 @@ module.exports = {
 
     app.options.sassOptions.includePaths.push('node_modules');
 
-    app.import('node_modules/uikit/dist/js/uikit.min.js');
+    app.import('vendor/uikit.min.js');
     app.import('node_modules/uikit/dist/js/uikit-icons.min.js');
```

Comment:
Why did you leave out the icons? I think if we support fastboot (which is definitely a good idea) we should support all of the features.

Reply:
I left icons alone because it does not throw an error under fastboot. Considering #12, which seems like a good idea, it is probably best to continue handling them separately.

Reply:
I'd still prefer adding it the same way for consistency since it doesn't hurt..

Reply:
A solution for #12 is implemented in #13. Take a look if you want and tell me your opinion!

Reply:
I really like what you did in #13 . Though that might cause a pretty significant rebase/refactor of this PR. Might be best to close this one so we can start fresh. :)

I hit a bit of a block on the testing last week. I haven't had a chance to dive back in. I got the ember-fastboot-addon-test set up on a no-dependency addon just fine. But I get errors with ember-uikit. Though I'm using my fork in a fastboot env without error. I can't remember all the errors. Each Node version (4, 6, and 7) all had different errors. 

When I left it, I was starting to suspect something screwy with my local system. I should have some time at the end of the week to work it out.

Reply:
Alright so we'll close this one in favor of a new one. I'll try to free some time to check out fastboot compatibility as well!

Thanks for your review and your contribution to this addon!